consensus/__main__.py

```diff
@@ -11,24 +10,20 @@
 
 def main():
     # Get data from config
-    config = ConfigParser('config/config.txt')  # To run by pressing play in pycharm, use ../config/config.txt
+    config = ConfigParser('../config/config.txt')  # To run by pressing play in pycharm, use ../config/config.txt
```

Comment:
Personally, I like using os.path.join() for input / output / data paths so operating systems do not pose a problem. Can be ignored.

Reply:
Actually this like of code is a bug now I see it. I shouldn't have changed this haha. It is a good suggestion. I might use it in future and maybe set the home directory of the code in the config so I won't have to change this line anymore when I want to run it in PyCharm.